supplementary/alica_ros1/alica_ros_turtlesim/libalica-turtlesim/src/FourCornersSummand.cpp

```diff
@@ -0,0 +1,63 @@
+#include "FourCornersSummand.h"
+#include "engine/blackboard/Blackboard.h"
+#include "engine/planselector/IAssignment.h"
+#include "turtle_interfaces.hpp"
+#include <fstream>
+#include <iostream>
+
+namespace turtlesim
+{
+
+FourCornersSummand::FourCornersSummand(double weight)
+        : USummand(weight)
+{
+}
+
+FourCornersSummand::~FourCornersSummand() {}
+
+/*
+    The goal of this summand is to priotize agent to pick up corner task which is closest to it.
```

Comment:
spelling, consider seeing if there is an extension for it, maybe that runs spell check only on comments